WooCommerce/Classes/Styles/Style.swift

```diff
@@ -88,10 +89,13 @@ class DefaultStyle: Style {
     let thinCaptionFont                 = DefaultStyle.fontForTextStyle(.caption1,
                                                                         weight: .thin,
                                                                         maximumPointSize: DefaultStyle.maxFontSize)
+    let footerLabelFont                 = DefaultStyle.fontForTextStyle(.footnote,
+                                                                        weight: .regular,
+                                                                        maximumPointSize: DefaultStyle.maxFontSize)
```

Comment:
non-blocking suggestion: I noticed that the footer font size is fixed while the device font size is updated without leaving the screen, and it's from having the font as a constant. I tried making it a computed variable and it updates automatically on device font size change (shown in screenshots below). Though, it's quite an edge case which users rarely would do, and if we do care about this case we can update for all font properties in a separate PR.

device font size | small | large
--- | --- | ---
when `footerLabelFont` is a constant | ![Simulator Screen Shot - iPhone 5s - 2019-07-25 at 11 08 26](https://user-images.githubusercontent.com/1945542/61843190-60f97280-aecd-11e9-8df8-1a4c13a3d339.png) | ![Simulator Screen Shot - iPhone 5s - 2019-07-25 at 11 07 44](https://user-images.githubusercontent.com/1945542/61843189-60f97280-aecd-11e9-87c3-1c8e5f21c4f4.png)
when `footerLabelFont` is a computed variable | ![Simulator Screen Shot - iPhone 5s - 2019-07-25 at 11 10 11](https://user-images.githubusercontent.com/1945542/61843211-78386000-aecd-11e9-9177-ad8471249bcf.png) | ![Simulator Screen Shot - iPhone 5s - 2019-07-25 at 11 10 08](https://user-images.githubusercontent.com/1945542/61843218-7cfd1400-aecd-11e9-97b5-563968a61416.png)